tests/conftest.py

```diff
@@ -1,9 +1,8 @@
 """
 Fixtures for testing.
 """
-# pylint: disable=redefined-outer-name, invalid-name, W0611
+# pylint: disable=redefined-outer-name, invalid-name, W0611, too-many-lines
```

Comment:
lol, @samredai please start writing smaller PRs. I bet in Java this lint error would come up in the Hello World app.

Reply:
Haha, I did feel awkward putting that there. It's because of the long list of examples I added to the conftest. Thanks for keeping me honest here, I moved them to an examples.py file and now both files are under the limit, allowing me to remove this. 🥳